hacking/aws_config/test-policies.yml

```diff
@@ -0,0 +1,60 @@
+# Usage: ansible-playbook test-policies.yml -vv
+#
+- hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: 'env python'
+
+    aws_region: "{{ region | default('us-east-1') }}"
+    iam_group_prefix: "{{ group_prefix | default('ansible-integration') }}"
+    iam_policy_prefix: "{{ policy_prefix | default('AnsibleIntegratation') }}"
+    aws_account_id: "{{ test_account_id | default('123456789012') }}"
+    ci_policies: '../../aws/policy/*.yaml'
+    test_policies: 'test_policies/*.yaml'
+
+  tasks:
+    - name: 'Check size of policies'
+      with_fileglob:
+        - '{{ test_policies }}'
+        - '{{ ci_policies }}'
+      set_fact:
+        policy: "{{ lookup('template', item) | from_yaml | to_json }}"
+      changed_when:
+        - policy_size | length > 9216
+      register: policy_size
+
+    - name: 'Fail if policy is too large'
+      vars:
+        _query: 'results[?changed].item'
+        oversized_policies: "{{ policy_size | json_query(_query) | list }}"
+      debug:
+        var: oversized_policies
+      failed_when:
+        - policy_size is changed
+
+    - name: Create location for policies
```

Comment:
This bit was me starting to play with using AWS tooling to validate the policies, so I needed a compiled copy.  I'm good with this just being removed, when I commented it out it generated extra linting warnings, so I just left it there in case you had a way to test the policies.

Reply:
What kind of tooling?  Policy Validator?  It might be interesting to provide something to do that, but obviously we don't have a module for that now.

Reply:
AWS CLI allows you to upload a policy to their simulation tool.  It's not as fussy (or wasn't last I checked) as IAM itself, but it's better than nothing.

Same idea that I was using in https://github.com/ansible/ansible/pull/67581